x-pack/plugins/monitoring/public/alerts/alert_form.test.tsx

```diff
@@ -165,19 +165,6 @@ describe('alert_form', () => {
       const alertTypeSelectOptions = wrapper.find('[data-test-subj="selectedRuleTypeTitle"]');
       expect(alertTypeSelectOptions.exists()).toBeTruthy();
     });
-
-    it('should update throttle value', async () => {
```

Comment:
We don't need to migrate this test to the new Action Form, because the NotifyWhen component no longer tracks the `throttle` value internallly.